specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.4-preview.1/keys.json

```diff
@@ -1352,6 +1354,14 @@
               {
                 "value": "oct-HSM",
                 "description": "Octet sequence (used to represent symmetric keys) which is stored the HSM."
+              },
+              {
+                "value": "OKP",
+                "description": "Octet key pair (https://tools.ietf.org/html/rfc8037)"
+              },
+              {
+                "value": "OKP-HSM",
+                "description": "Octet key pair (https://tools.ietf.org/html/rfc8037) with a private key which is stored in the HSM."
```

Comment:
While the descriptions seem fine, I'm wondering if this support extends to Key Vault or only Managed HSM? In the latter case, isn't this description always true then?

Reply:
This key type is only supported for Managed HSM. So, OKP and OKP-HSM are synonymous and both use the HSM for storing the keys. I have no insights into whether Key Vault will support this key type eventually. 

I followed the example of the `oct` key type which AFAIK is also only supported by Managed HSM.